corehq/apps/export/models/new.py

```diff
@@ -253,7 +263,12 @@ class ExportDataSchema(Document):
     An object representing the things that can be exported for a particular
     form xmlns or case type. It contains a list of ExportGroupSchema.
     """
+    domain = StringProperty()
+    created_on = DateTimeProperty(default=datetime.utcnow())
```

Comment:
`default` should be a callable or a static value otherwise you'll get the value from when the class was first loaded.